src/Nethermind/Nethermind.JsonRpc/JsonRpcConfig.cs

```diff
@@ -53,6 +53,6 @@ public int WebSocketsPort
         public string[] EngineEnabledModules { get; set; } = ModuleType.DefaultEngineModules.ToArray();
         public int MaxBatchSize { get; set; } = 1024;
         public long? MaxBatchResponseBodySize { get; set; } = 30.MB();
+        public bool UseMinGasPriceInEstimates { get; set; } = false;
```

Comment:
One small thing, I would rename it to `ForceMinGasPriceInEstimates` and move to `IBlocksConfig`

Reply:
Hey @Andrew-Pohl can you do this change?